src/main/java/ch/uzh/ifi/hase/soprafs21/service/UserService.java

```diff
@@ -123,10 +124,16 @@ public boolean loginOrRegisterUser(GoogleIdToken.Payload payload, String refresh
     }
 
     public boolean isRequesterAndAuthenticatedUserTheSame(Long senderId) {
-        String authenticatedUserEmail = (String) SecurityContextHolder.getContext().getAuthentication().getPrincipal();
-        User authenticatedUser = userRepository.findByEmail(authenticatedUserEmail).orElseThrow();
-        User sender = userRepository.getOne(senderId);
-        return authenticatedUser.equals(sender);
+        //String authenticatedUserEmail = (String) SecurityContextHolder.getContext().getAuthentication().getPrincipal();
+        //User authenticatedUser = userRepository.findByEmail(authenticatedUserEmail).orElseThrow();
+        //User sender = userRepository.getOne(senderId);
+        //return authenticatedUser.equals(sender);
```

Comment:
If it is not longer used, you can delete it

Reply:
updated